ReactWindows/ReactNative.Shared/Bridge/Queue/MessageQueueThread.cs

```diff
@@ -151,9 +153,9 @@ public DispatcherMessageQueueThread(string name, Action<Exception> handler)
             {
                 _actionSubject = new Subject<Action>();
                 _actionObserver = _actionSubject;
-#if NET46
+#if WINDOWS_UWP
```

Comment:
I realize I'm making the diff larger by switching these, but they were inconsistent with the ones we've already done.